cmds/pwd/pwd.go

```diff
@@ -24,8 +23,11 @@ import (
 )
 
 var (
-	logical  = flag.Bool("L", true, "Follow symlinks") // this is the default behavior
-	physical = flag.Bool("P", false, "Don't follow symlinks")
+	// This is the default. Setting it to false doesn't do anything in GNU
+	// or zsh pwd, because you just can't even set it to false.
+	_ = flag.Bool("L", true, "don't follow any symlinks")
+
+	physical = flag.Bool("P", false, "follow all symlinks (avoid all symlinks)")
```

Comment:
what? 
follow all symlinks (avoid all symlinks)

this is confusing to me. Which is it?

Reply:
It's follow all symlinks. The man page for pwd helpfully calls that "avoid all symlinks", but what that means is follow all symlinks. Maybe I should just remove that?